samples/msgext-ai-doc-compliance-checker/README.md

```diff
@@ -0,0 +1,130 @@
+      AZURE_CONTAINER_NAME={{Azure_Container_Name}}
+      CHECKLIST_NAME={{Name of the Policy Guideline document uploaded in Azure Blob}}
+   ```
+
+## Run the app (Using Teams Toolkit for Visual Studio Code)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio Code.
+
+1. Ensure you have downloaded and installed [Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview)
+1. Install the [Teams Toolkit extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+1. Select **File > Open Folder** in VS Code and choose this samples directory from the repo
+1. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps
+1. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
+1. In the browser that launches, select the **Add** button to install the app to Teams.
+
+> If you do not have permission to upload custom apps (sideloading), Teams Toolkit will recommend creating and using a Microsoft 365 Developer Program account - a free program to get your own dev environment sandbox that includes Teams.
+
+## Running the sample
+
+1) **Add Compliance Checker App:**
+![ComplianceCheckerInstallApp](Images/1.ComplianceCheckerInstallApp.png)
+
+1) **Search for the Compliance Checker app from the Messaging Extensions:**
+![ComplianceCheckerSearchApp](Images/2.ComplianceCheckerSearchApp.png)
+
+1) **Select and add the Compliance Checker App:**
+![ComplianceCheckerSelectApp](Images/3.ComplianceCheckerSelectApp.png)
+
+1) **Search for the file name of the proposal document to compare against the policy document:**
+![ComplianceCheckerSearchFileName](Images/4.ComplianceCheckerSearchFileName.png)
+
+1) **Compliance Assessment Result. The proposal document is checked against the guideline document and the results are shown in an adaptive card:**
+![ComplianceCheckerSelectResults](Images/5.ComplianceCheckerSelectResults.png)
+
+1) **Compliance Checker Results:**
+![6.ComplianceCheckerResults](Images/6.ComplianceCheckerResults.png)
+
+1) **Click on the arrow to expand the card and get contextual reasoning for the complaince status being Yes or No:**
+![7.ComplianceCheckerYesResults](Images/7.ComplianceCheckerYesResults.png)
+
+![8.ComplianceCheckerNoResults](Images/8.ComplianceCheckerNoResults.png)
+
+### Test the app in Copilot for Microsoft 365
+
+1) **Open Microsoft Teams, navigate to the Copilot section from the left-hand menu, look for the "Plug-ins" section on the right side of the screen, and use the search bar within the Plug-ins menu to search for "Compliance Checker":**
+![10.Copilot](Images/10.Copilot.png)
+
+1) **Enable Compliance Checker app:**
+![11.CopilotEnable](Images/11.CopilotEnable.png)
+
+1) **Type Prompt: "Using Compliance Checker, evaluate Acme Suppliers Proposal for compliance" and click the send icon:**
```

Comment:
`Using Compliance Checker, evaluate Acme Suppliers Proposal for compliance`

Reply:
Fixed